anvil/src/udev.rs

```diff
@@ -76,8 +75,14 @@ impl AsRawFd for SessionFd {
     }
 }
 
+struct UdevOutputMap {
+    pub device_id: dev_t,
+    pub crtc: crtc::Handle,
+    pub output_name: String,
+}
```

Comment:
I get the feeling, that we might want to add userdata to the output-map at some point. But that exercise can be left to be done later, this is sufficient.

Reply:
Yes, i considered this, but wanted a simple solution for now cause i expect that the output will change a bit when implementing HiDPI. Maybe i will add a function to get the (currently) private `Output` struct in `window_map`, then
it would be easy to access a `UserDataMap` from the `Output`.